src/ProgressOnderwijsUtils/Data/ParameterizedSqlObjectMapper.cs

```diff
@@ -516,12 +521,10 @@ public bool Equals(ColumnOrdering other)
                 static ByMetaObjectImpl()
                 {
                     var writablePropCount = 0;
-                    var debug = "";
                     foreach (var mp in metadata) { //perf:no LINQ
-                        if (mp.CanWrite && SupportsType(mp.DataType) || SupportsTypeByCustomCreateMethod(mp.DataType)) {
+                        if (IsWritableSimpletype(mp) || SupportsTypeByCustomCreateMethod(mp)) {

```

Comment:
Zou het kunnen kloppen dat deze methods altijd alleen in deze combinatie aangeroepen worden?

Dat zou logisch zijn; en als dat zo is: evt. fuseren?